ls1intum/Artemis
# Feature/markdown editor codeblock syntax highlighting

Add a new markdown command for code blocks and showdown-highlight to properly highlight code in the markdown editor

### Checklist

### Steps for Testing
1. Add code into a problem statement using the new code block command, e.g.
2. Check in the preview and on the exercise details page that the code is displayed properly and keywords are hightlighted
3. Try out code for different programming languages, e.g. Java, Python, ...

<img width="281" alt="image" src="https://user-images.githubusercontent.com/744067/105252339-85dfac00-5b7d-11eb-870e-58c15887d251.png">

<img width="300" alt="image" src="https://user-images.githubusercontent.com/744067/105252362-8c6e2380-5b7d-11eb-89e4-c26c22e8c175.png">
